docker-compose.yml

```diff
@@ -10,12 +10,20 @@ services:
     restart: unless-stopped
     environment:
       - ROOT_PATH=/backend
-      - HYPERTEXT_PROTOCOL=https://
-      - VOTING_SERVICE_PATH=team17-21.studenti.fiit.stuba.sk/gateway/voting-service-api
-      - STATE_VECTOR_PATH=team17-21.studenti.fiit.stuba.sk/gateway/statevector
-      - VOTING_PROCESS_MANAGER_HOST=team17-21.studenti.fiit.stuba.sk
-      - VOTING_PROCESS_MANAGER_HOST_SOCKET_PATH=gateway/voting-process-manager-api/ws/socket.io
-      - VOTING_PROCESS_MANAGER_PATH=team17-21.studenti.fiit.stuba.sk/gateway/voting-process-manager-api
+      - VOTING_SERVICE_PATH=http://host.docker.internal:8080/voting-service-api
+      # - VOTING_SERVICE_PATH=https://team17-21.studenti.fiit.stuba.sk/gateway/voting-service-api
+      - STATE_VECTOR_PATH=http://host.docker.internal:8080/statevector
+      # - STATE_VECTOR_PATH=team17-21.studenti.fiit.stuba.sk/gateway/statevector
+
+      - VOTING_PROCESS_MANAGER_HOST=http://host.docker.internal:8080
+      - VOTING_PROCESS_MANAGER_HOST_SOCKET_PATH=voting-process-manager-api/ws/socket.io
+      - VOTING_PROCESS_MANAGER_PATH=http://host.docker.internal:8080/voting-process-manager-api
+      # - VOTING_PROCESS_MANAGER_HOST=https://team17-21.studenti.fiit.stuba.sk
+      # - VOTING_PROCESS_MANAGER_HOST_SOCKET_PATH=gateway/voting-process-manager-api/ws/socket.io
+      # - VOTING_PROCESS_MANAGER_PATH=https://team17-21.studenti.fiit.stuba.sk/gateway/voting-process-manager-api
```

Comment:
hej no, je to pekné, že si to človek vie spustiť s tým remote. Ale namiesto zakomentovania navrhujem vytvoriť nový docker-compose.remote.yml, kde budú tie remote adresy.

Reply:
A tým, že si z kódu zmazal tie http veci, treba upraviťenvs aj v docker-compose prod, stage a testbackend.

Reply:
jj ten state vector bola moje mrdnutost. skus ted